hugo/layouts/company/single.html

```diff
@@ -495,6 +495,16 @@ <h4 class="text-dark title-line line-yellow mt-3">Fernanda Gomes</h4>
                     </div>
                 </div>
             </div>
+            <div class="col-sm-6 col-lg-3 mb-4">
+                <div class="card white">
+                    <img class="w-100" src="assets/img/team/alicia.png" alt="">
```

Comment:
Hi @aliciakozlowski :) I know none of the team photo images has the alt="" attribute set, but it would be nice - for accessibility sake - for you to add it "Alicia Kozlowski". If you want/have the time to go further you can do it for all team member photos, please :)